test/e2e/data/e2e_conf.yaml

```
@@ -213,12 +213,12 @@ variables:
   INIT_WITH_BINARY_V1BETA1: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.2.0/clusterctl-{OS}-{ARCH}"
   # INIT_WITH_KUBERNETES_VERSION are only used by the clusterctl upgrade test to initialize
   # the management cluster to be upgraded.
-  INIT_WITH_KUBERNETES_VERSION: "v1.30.8"
+  INIT_WITH_KUBERNETES_VERSION: "v1.31.0"
   EXP_BOOTSTRAP_FORMAT_IGNITION: "true"
   EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION: "true"
   GC_WORKLOAD: "../../data/gcworkload.yaml"
   CLASSICELB_TEST_KUBERNETES_VERSION_FROM: "v1.29.9"
-  CLASSICELB_TEST_KUBERNETES_VERSION_TO: "v1.30.8"
+  CLASSICELB_TEST_KUBERNETES_VERSION_TO: "v1.31.0"
```

This is causing [an error](https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_cluster-api-provider-aws/5447/pull-cluster-api-provider-aws-e2e/1925273442618183680#):

```
Message: "Forbidden: cannot update Kubernetes version from v1.29.9 to v1.31.0",
```
Let's not bump this, as this is required to test the v1.29 -> v1.30 upgrade behaviour for the deprecated Classic ELBs.
```suggestion
  CLASSICELB_TEST_KUBERNETES_VERSION_TO: "v1.30.8" # keep this as v1.30.z
```